documentation/providers/loopia.md

```diff
@@ -0,0 +1,214 @@
+
+This plugin is based on API documents found at 
+[https://www.loopia.com/api/](https://www.loopia.com/api/)
+and by observing API responses. Hat tip to Github @hazzeh whose code for the
+LEGO Loopia implementation was helpful.
+
+Sadly the Loopia API has some problems:
+* API calls are limited to 60 calls per minute.  If you go above this,
+  you will have to wait before you can make changes.
+* When rate-limited, you will not receive a single HTTP
+  error: The errors propagate from the back-end, with no headers, or
+  Retry-After or anything useful.
+* There are no guarantees of idempotency from their API.
+
+## Unimplemented API methods
+
+ * `removeDomain` is not implemented for safety reasons. Should you wish to remove
+a domain, do so from the Loopia control panel.
+ * `addDomain`
+ * `transferDomain` (to Loopia)
+
+This effectively means that this plugin does not access registrar functions.
+
+## Errors
+
+You may occasionally see this error
+```text
+HTTP Post Error: Post "https://api.loopia.se/RPCSERV": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
+```
+
+The API endpoint didn't answer. Try again. 🤷
+
+
+## Configuration
+
+To use this provider, add an entry to `creds.json` with `TYPE` set to `LOOPIA`
+along with your Loopia API login credentials.
+
+Example:
+
+{% code title="creds.json" %}
+```json
+{
+  "loopia": {
+    "TYPE": "LOOPIA",
+    "username": "your-loopia-api-account-id@loopiaapi",
+    "password": "your-loopia-api-account-password",
+    "debug": "true" // Set to true for extra debug output. Remove or set to false to prevent extra debug output. 
+  }
+}
+```
+{% endcode %}
+
+### Variables
+
+* `username` - string - your @loopiaapi created username
+* `password` - string - your loopia API password
+* `debug` - string - Set to true for extra debug output. Remove or set to false to prevent extra debug output. 
+* `rate_limit_per` - string - See [Rate Limiting](#rate-limiting) below.
+* `region` - string - See [Regions](#regions) below.
+* `modify_name_servers` - string - See [Modify Name Servers](#modify-name-servers) below.
+* `fetch_apex_ns_entries` - string - See [Fetch NS Entries](#fetch-apex-ns-entries) below.
+
+There is no test endpoint. Fly free, grasshopper.
+
+Turning on debug will show the XML requests and responses, and include the
+username and password from your `creds.json` file. If you want to share these, 
+like for a Github issue, be sure to redact those from the XML.
+
+### Fetch Apex NS Entries
+
+... or use locally hard-coded variables:
+
+```go
+  defaultNS1       = "ns1.loopia.se."
+  defaultNS2       = "ns2.loopia.se."
+```
+
+API calls to loopia can be expensive time-wise. Set this to "false" (off) to
+skip the API call to fetch the apex (`@`) entries, and use Loopia's default NS
+servers.
+
+This setting defaults to "true" (on).
+
+### Modify Name Servers
+
+Enabling this setting to "true" (on) allows you to modify NS entries.
```

Comment:
It isn't clear that "this setting" refers to `modify_name_servers`.  Change to:

Setting `modify_name_servers` in `creds.json' to `true`  (on) allows...

Reply:
Fixed. Thanks!